plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/messages.properties

```diff
@@ -0,0 +1,3 @@
+FIELD_EDITOR_ANALYTICS_DISCLAIMER=Your use of this plugin is subject to the Apache 2.0 software license and the <a href="privacy-policy">Google Privacy Policy</a>.
+FIELD_EDITOR_ANALYTICS_GROUP_TITLE=Usage Statistics
+FIELD_EDITOR_ANALYTICS_OPT_IN_TEXT=&Share anonymous usage statistics of Google Cloud Plugins with Google
```

Comment:
The standard Eclipse term is _plug-ins_ with a hyphen. Bizarre but it is what it is.

That said, perhaps we could just use _Google Cloud Tools_ and avoid the issue entirely :-)